.github/workflows/pull_request.workflow.yml

```diff
@@ -56,6 +34,28 @@ jobs:
       - uses: cedx/setup-dart@v2
       - uses: ./.github/actions/unit-tests
 
+  doc-dead-links:
+    name: Check dead-links
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Extract references from context
+        shell: bash
+        id: extract-refs
+        run: |
+          echo "::set-output name=version::$(git describe --abbrev=0 --tags | cut -d. -f 1)"
+          echo "::set-output name=repo::$(echo $GITHUB_REPOSITORY | cut -d/ -f 2)"
+      - uses: convictional/trigger-workflow-and-wait@v1.3.0
+        with:
+          owner: kuzzleio
+          repo: documentation
+          github_token: ${{ secrets.ACCESS_TOKEN_CI }}
+          workflow_file_name: dead_links.workflow.yml
+          ref: gha-ci # TODO put develop here
```

Comment:
PR could be created from the master branch (hotfixes...)

Reply:
Maybe you could add step to get the proper branch to use. `GITHUB_BASE_REF` is available to get the base branch of the PR 😉